Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.7.4
+
+EXPOSE 8080
+RUN \
+  curl -fsSLO https://get.docker.com/builds/Linux/x86_64/docker-latest.tgz && \
+  tar --strip-components=1 -xvzf docker-latest.tgz -C /usr/local/bin
+RUN mkdir -p /go/src/go.uber.org/sally
+WORKDIR /go/src/go.uber.org/sally
+ADD glide.yaml glide.lock /go/src/go.uber.org/sally/
+RUN go get -v github.com/Masterminds/glide && glide install
```

Comment:
This is actually done on purpose for caching. Without this, things like `make docker-test` will take forever. Note that if glide.yaml or glide.lock change, the docker build's cache is no longer valid, so `glide install` will run again.